templates/blog/_post.html

```diff
@@ -1,6 +1,22 @@
-<article>
-  <h1>{{ article.title.rendered|safe }}</h1>
-  <div>
-    {{ article.content.rendered|safe }}
-  </div>
-</article>
\ No newline at end of file
+<div>
+  <article class="p-strip">
+    <div class="row">
+      <div class="col-7 p-post">
+        {{ article.content.rendered|safe }}
+        
+        {% for related_article in related_articles %}
+          <hr class="p-separator" />
+          {% include 'blog/_media-object.html' %}
+          {% if loop.last %}
+            <hr class="p-separator__mobile" />
+          {% endif %}
```

Comment:
I think this block can be removed and added after the `for` block. Like this we avoid a if for every iteration in the loop